checkers/Board.py

```diff
@@ -224,6 +312,16 @@ def move_piece(self, mouse_x, mouse_y):
                 self.selected_piece[1]].selected = False
             self.grid[selected_row][selected_col] = self.grid[
                 self.selected_piece[0]][self.selected_piece[1]]
+            
+            # update the position attribute of the piece
+            self.grid[selected_row][selected_col].position = (selected_row, selected_col)
+            
+            # make changes on the status of a piece based on its moved position
+            if self.grid[selected_row][selected_col].position[0] == 0 and player_turn == -1:
+                self.grid[selected_row][selected_col].status = 'king'
+            if self.grid[selected_row][selected_col].position[0] == 7 and player_turn == 1:
```

Comment:
I think you can use only one _if_ statement here, let another one as _elif_.

Reply:
Agree.